<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rabbit_rad">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM EULJIRO TTF</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rabbit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bear_rad">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM EULJIRO TTF</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choice_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>640</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="choiceBackground_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="easteregg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>700</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon</family>
      <pointsize>3</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(160, 80, 0);</string>
    </property>
    <property name="text">
     <string>^ . , . ^</string>
    </property>
   </widget>
   <zorder>choiceBackground_label</zorder>
   <zorder>rabbit_rad</zorder>
   <zorder>bear_rad</zorder>
   <zorder>choice_btn</zorder>
   <zorder>easteregg</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
